portafly/src/components/util/AlertsContext.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useContext } from 'react'
+import {
+  AlertGroup,
+  AlertActionCloseButton,
+  Alert,
+  AlertVariant
+} from '@patternfly/react-core'
+
+interface IAlert {
+  id: string
+  variant: keyof typeof AlertVariant
+  title: string
+  timeout: NodeJS.Timeout
+}
+
+type IAlertsContext = { addAlert: (alert: Omit<IAlert, 'timeout'>) => void }
+const AlertsContext = React.createContext<IAlertsContext>({ addAlert: () => {} })
+
+const AlertsProvider: React.FunctionComponent = ({ children }) => {
+  const [alerts, setAlerts] = React.useState<IAlert[]>([])
+
+  const removeAlert = (id: string) => {
+    setAlerts((prevAlerts) => prevAlerts.filter((a) => a.id !== id))
+  }
+
+  const addAlert = (alert: Omit<IAlert, 'timeout'>) => {
+    const timeout = setTimeout(() => removeAlert(alert.id), 8000)
+    setAlerts((prevAlerts) => [...prevAlerts, { ...alert, timeout }])
+  }
+
+  const CloseButton = ({ id, timeout }: Pick<IAlert, 'id' | 'timeout'>) => (
+    <AlertActionCloseButton
+      title="button title"
+      variantLabel="variant alert"
```

Comment:
These will be located eventually, right?

Reply:
If you mean localized 😁, yes I missed it